### ISSUE TITLE
Add Lottie as Image Source

### ISSUE THREAD

**Issue by jrahma** [xamarin/Xamarin.Forms#8998]
Hi,

It will be great if you provide a feature to add a a Lottie file as Image Source, something like this:

`<Image Source="MyLottieFile" />
`

Thanks,
[Jassim](https://www.softnames.com)

**Comment by bill2004158** [comment]
@jrahma 

https://github.com/Baseflow/LottieXamarin
maybe that is what you need?

**Comment by jrahma** [comment]
Yes but it will be better to get it through Image control just like the animated GIF support which was added in latest version

**Comment by xamarindevelopervietnam** [comment]
++

**Comment by jfversluis** [comment]
This will not happen anymore for Xamarin.Forms. If this is still important to you, make sure to check the [.NET MAUI repo](https://github.com/dotnet/maui) and see if it's already on the roadmap or open an issue with a detailed feature request. Thanks!
